### PR TITLE
chore(codestyle): apply prettier to packages

### PR DESCRIPTION
**Pull request by wermanoid**
## A picture tells a thousand words
There is a `prettier` and `.prettierrc.cjs` in repo

## Before this PR
It was not applied to code, so any change leads to massive file changes

## After this PR
Just applied code formatting for packages, so less annoying and unexpected file changes

## How to reproduce
run `pnpm prettier --write --cache packages`

### CODE REVIEW

**Comment by wermanoid** [comment]
Slightly unexpected, but I don't know why code in packages was not aligned with configured tool for code styling. In my setup, when formatting is applied on every Ctrl+S it leads to massive changes and huge git diff. So fixing style issue and will add more PR's later

**Comment by guillaume-chervet** [comment]
Thank you very much @wermanoid for your PullRequest.

I will merge this PullRequest before https://github.com/AxaFrance/oidc-client/pull/1410 and also this one before https://github.com/AxaFrance/oidc-client/pull/1402.

It may impact your PullRequest.

**Comment by wermanoid** [comment]
I will keep it up-to-date, so merge whatever you need :) no problems

**Comment by jafin** [comment]
@wermanoid  I would suggest reviewing the prettier config to align to the existing project source, so yeah tab should be replaced with whatever the most common existing pattern is in the project, which appears to be `2 spaces`  not tab.

i.e. treat the project's existing approach as preferred and adjust prettierrc to conform, vs prettier being the point of truth now.
I think I may of added prettier as a placeholder a ways back, and not correctly aligned to project style.

Also, there appears to be a disconnect between ESlint running prettier rules and running prettier externally.   Ideally, ESlint should be reading the prettier config and if request autofix with `eslint --fix`  . So I think there is something amiss with the eslint-> prettier config. 

@guillaume-chervet does this sound ok?

**Comment by jafin** [comment]
@wermanoid  @guillaume-chervet 
I found the issue with eslint not running prettier.   If you commit #1419 it will ensure eslint runs prettier.  
I haven't made changes to prettier config or applied the prettier autofix, as this is what this PR is for. 
`pnpm run lint` will now show the prettier issues.
`pnpm run lint --fix` will autofix them

**Comment by wermanoid** [comment]
Updated after all eslint fixes

File: packages/oidc-client-service-worker/src/utils/__tests__/testHelper.ts [hunk 1]
```diff
@@ -130,16 +126,15 @@ class OidcConfigBuilder {
...
     demonstratingProofOfPossessionConfiguration: null,
     demonstratingProofOfPossessionOnlyWhenDpopHeaderPresent: false,
-    allowMultiTabLogin: true,
   };
 
   public withTestingDefault(): OidcConfigBuilder {
     this.oidcConfig.configurationName = 'test';
     this.oidcConfig.tokens = new TokenBuilder().withNonExpiredToken().build();
     this.oidcConfig.status = 'NOT_CONNECTED';
-    this.oidcConfig.state = { tab1: 'state' };
```
Comment (guillaume-chervet) [inline packages/oidc-client-service-worker/src/utils/__tests__/testHelper.ts:140]: I think the correct new version is { tab1: 'state' }; in the whole file @wermanoid

**Comment by guillaume-chervet** [comment]
@jafin I do not code a lot in Javascript/typescript these days (unfortunately). 

This configuration seems OK to you?

File: packages/oidc-client-service-worker/src/utils/__tests__/testHelper.ts (hunk 1 above)
Comment (guillaume-chervet) [reply packages/oidc-client-service-worker/src/utils/__tests__/testHelper.ts:140]: I think it explain why unit tests are failing @wermanoid

Comment (wermanoid) [reply packages/oidc-client-service-worker/src/utils/__tests__/testHelper.ts]: Ok, updated from clear main branch. Should not introduce anything unexpected now

**Review by guillaume-chervet — APPROVED** [review]
Thank you very much @wermanoid I merge like it and I will fixe the few Types erros in another PullRequest.

**Pull request merged**